extensions/adobe/experience/intelligentServices/profile-journeyai-engagementscores.schema.json

```diff
@@ -0,0 +1,103 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/intelligentServices/profile-journeyai-engagementscores",
+  "title": "Intelligent Services JourneyAI Engagement Scores",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/context/profile"],
+  "description": "This mixin contains JourneyAI Engagement Scores, which provides individual customer's propensities to engage with a message.",
+  "definitions": {
+    "journeyai-engagement-scores": {
+      "properties": {
+        "https://ns.adobe.com/experience/intelligentServices/journeyAI": {
+          "title": "journeyAI",
+          "description": "JourneyAI intelligent services provides various insights about message deliveries along customer's journey",
+          "type": "object",
+          "isRequired": false,
+          "properties": {
+            "https://ns.adobe.com/experience/intelligentServices/journeyAI/engagementScores": {
+              "title": "engagementScores",
+              "description": "JourneyAI Engagement Scores provide likelihoods of a customer engaging with a newly delivered message.",
+              "type": "object",
+              "isRequired": false,
+              "properties": {
+                "https://ns.adobe.com/experience/intelligentServices/journeyAI/emailEngagementScores": {
+                  "title": "emailEngagementScores",
+                  "description": "Engagement scores for email channel messages",
+                  "type": "object",
+                  "isRequired": false,
+                  "properties": {
+                    "https://ns.adobe.com/experience/intelligentServices/journeyAI/emailClickEngagementScore": {
+                      "title": "emailClickEngagementScore",
+                      "description": "probability for a customer to click on links in an email",
+                      "type": "number",
+                      "isRequired": false,
```

Comment:
We don't use isRequired flag at the property level. Please remove all across

Reply:
Done